src/plugins/expressions/common/expression_functions/specs/tests/font.test.ts

```diff
@@ -39,163 +50,117 @@ describe('font', () => {
   describe('args', () => {
     describe('size', () => {
       it('sets font size', () => {
-        const result = fn(null, { size: 20 });
+        const result = fn(null, { ...args, size: 20 });
         expect(result).toMatchObject({
           spec: {
             fontSize: '20px',
           },
         });
         expect(result.css).toContain('font-size:20px');
       });
-
-      it('defaults to 14px', () => {
-        const result = fn(null);
-        expect(result).toMatchObject({
-          spec: {
-            fontSize: '14px',
-          },
-        });
-        expect(result.css).toContain('font-size:14px');
-      });
```

Comment:
One nit here -- it would be cool to add an integration test for `font` that actually validates the correct default value is applied via `theme`, based on whether or not you have set the appropriate variable.

Reply:
that won't be possible due to the way interpreter works:
- before running any function (font in this case) it will check if any arguments are missing and if defaults are provided for them
- it will evaluate the default value (interpret and execute), so if default is expression it will run it
- it will provide the result of expression as an actual argument to our function (font)

here we are just testing the font function without interpreter and providing arguments in. to do integration test we would need interpreter service running (with the registries and everything) which i think is hard at the moment.